libs/akita-ng-entity-service/src/lib/types.ts

```diff
@@ -28,6 +28,7 @@ export type HttpConfig<Entity = any> = {
   params?: _HttpParams;
   headers?: _HttpHeaders;
   url?: string;
+  subResources?: string[];
```

Comment:
Why use an array, and why couple it to "resources". I think we can change it to:

`urlPostfix: string;`

`urlPostfix: '/some/3/page/3'`

Reply:
I agree. I did it in a form of an array because I had in my mind `router.navigate([])` but concatenating the parameters to the string is just as good and "urlPostfix" definitely sounds better. I'll change it.